draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,39 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update
+
+When a CID record is received with a different source address than the
```

Comment:
How about 

"When a datagram with a CID is received that has a source address different than the one currently associated with the DTLS connection, the receiver MUST NOT replace the source address currently stored in the association with the source address specified in the received datagram unless the following conditions are met:
 * The received datagram has been cryptographically verified using the DTLS record layer processing procedures. 
- The received datagram are "newer" (in terms of their epoch and sequence
  number) than the last datagram that successfully updated the source address.

Reply:
I like it, but I think using the term datagram instead of record when talking about crypto operations mixes layers.